Altis_Life.Altis/core/actions/fn_dropFishingNet.sqf

```diff
@@ -5,9 +5,9 @@
     Description:
     Drops a virtual fishing net from the boat.
 */
-private ["_fish","_type","_typeName"];
+private ["_type","_typeName"];
```

Comment:
private _type = "";
private "_typeName";
and remove default from the yucky switch true